sdk/src/test/java/io/dapr/client/DaprClientGrpcTest.java

```diff
@@ -971,7 +1081,10 @@ public synchronized void onSuccess(@NullableDecl T result) {
     public synchronized void onFailure(Throwable throwable) {
       assertFalse(wasCalled);
       wasCalled = true;
-      assertEquals(failure, throwable);
+      assertEquals(failure == null, throwable == null);
```

Comment:
Better assertions to validate exception. Forcing the equals() was too strong for this check (IMO).